maven/com/kaltura/KalturaClientLib/maven-metadata-local.xml

```diff
@@ -13,6 +13,7 @@
             <version>3.2.1</version>
             <version>3.2.2</version>
             <version>3.3.3</version>
+            <version>3.3.4</version>
```

Comment:
Consider to update also https://github.com/kaltura/server/blob/Lynx-12.11.0/api_v3/VERSION.php